presage/src/model/messages.rs

```diff
@@ -0,0 +1,13 @@
+use libsignal_service::prelude::Content;
+
+#[derive(Debug)]
+pub enum Received {
+    /// when the receive loop is empty, happens when opening the websocket for the first time
+    /// once you're done synchronizing all pending messages for this registered client.
+    QueueEmpty,
+
+    // got contacts synchronization
+    Contacts,
+
+    Content(Content),
```

Comment:
Content is 4 KiB in memory. That's why clippy complains about huge size difference on the variants. I think it makes sense to box it.